py-polars/polars/expr/datetime.py

```diff
@@ -390,6 +390,38 @@ def round(self, every: str | dt.timedelta | IntoExprColumn) -> Expr:
         every = parse_into_expression(every, str_as_lit=True)
         return wrap_expr(self._pyexpr.dt_round(every))
 
+    def replace(
+        self,
+        year: int | IntoExpr | None = None,
+        month: int | IntoExpr | None = None,
+        day: int | IntoExpr | None = None,
+        hour: int | IntoExpr | None = None,
+        minute: int | IntoExpr | None = None,
+        second: int | IntoExpr | None = None,
+        microsecond: int | IntoExpr | None = None,
+        ambiguous: Ambiguous | Expr = "raise",
```

Comment:
I suppose they should be keyword-only arguments.

Reply:
They are already, no?

Reply:
Done, thanks for the suggestion @eitsupi.

Reply:
Thanks!